Gemfile

```diff
@@ -166,7 +166,7 @@ group :development do
 
   # Simple command execution over SSH. Lightweight deployment tool.
   # Using our own version until https://github.com/mina-deploy/mina/pull/361 is merged.
-  gem 'mina', git: "git@github.com:samfundet/mina.git"
+  gem 'mina', git: "git://github.com/Samfundet/mina.git"
```

Comment:
According to https://git-scm.com/book/tr/v2/Git-on-the-Server-The-Protocols git:// seems to have no authentication or security whatsoever.
I thought it was a wrapper around https, but seems not so.
We should probably change it then?